tensorflow/go/README.md

```diff
@@ -118,6 +118,25 @@ from source.
     go test github.com/tensorflow/tensorflow/tensorflow/go
     ```
 
+### Generate Ops Wrapper
+
+Tensorflow ops wrappers in Go is generated by a script. Here is how to generate Go source files for Tensorflow ops.
+
+Prerequisites
+- Protocol Buffer 3.x
+- Downloading Tensorflow repository under GOPATH
+
+```
+# Gegerate Go source from protobuf files.
```

Comment:
This can be replaced by a single:

```
go generate github.com/tensorflow/tensorflow/tensorflow/go/op
```
(Doing so does the generation needed for the `genop` tool as well as actually generating wrappers.go)